docs/release-notes/release-notes-0.9.11.md

```diff
@@ -0,0 +1,176 @@
+Joinmarket-clientserver 0.9.11:
+=================
+
+<https://github.com/joinmarket-org/joinmarket-clientserver/releases/tag/v0.9.11>
+
+Please report bugs using the issue tracker at github:
+
+<https://github.com/joinmarket-org/joinmarket-clientserver/issues>
+
+Upgrading
+=========
+
+To upgrade:
+
+*Reminder: always back up and recreate your joinmarket.cfg file when doing the upgrade; this is to make sure you have the new default settings. In order to recreate it, rename the old joinmarket.cfg and run 'python3 wallet-tool.py generate' from the scripts folder.*
+
+(If you are upgrading from a version pre-0.7.0 please read the "Upgrading" section in [the 0.7.0 release notes](https://github.com/JoinMarket-Org/joinmarket-clientserver/blob/master/docs/release-notes/release-notes-0.7.0.md).)
+
+(If you are upgrading from a version pre-0.9.0 please read the [release notes](https://github.com/JoinMarket-Org/joinmarket-clientserver/blob/master/docs/release-notes/release-notes-0.9.0.md) to find out how about [fidelity bonds](../fidelity-bonds.md) are relevant to your use-case).
+
+First run the `install.sh` script as mentioned in the README. When prompted to overwrite the directory `jmvenv`, accept.
+
+Note that `./install.sh -?` will show the options for installation.
+
+Changes
+===============
+
+### Removal of Python 3.7 support, 3.8 or newer is required now
+
+Python 3.7 has been end of-life since June 2023 in it was blocker for some necessary dependency updates.
+
+Note that we currently only Python 3.8, 3.9, 3.10 and 3.11 are supported, not Python 3.12 and newer.
+
+* `7e045c9` Requires Python <3.12
+* `3d56837` Remove unnecessary check for Python 3.7+
+* `204f117` Drop Python 3.7 support
+
+### Move to modern packaging and source layout
+
+- Migrate to src-layout following https://setuptools.pypa.io/en/latest/userguide/package_discovery.html#src-layout to facilitate automatic package discovery and editable installs.
+- Create `joinmarket` namespace distribution package in line with https://packaging.python.org/en/latest/guides/packaging-namespace-packages/
+- Migrate to `pyproject.toml` file for project configuration instead of legacy `setup.py`
+
+* `c8eef50` Migrate to modern packaging and src layout
+
+### Performance improvements
+
+Lots of performance improvements made in wallet code that makes big impact especially with old wallets, with a lots of transaction history (in extreme cases time needed for some operations was improved from 40+ minutes to around 10 seconds).
+
+This involves caching some computed data in wallet, for cases when if there is some bug with this discovered, new configuration option `wallet_caching_disabled` is added to `joinmarket.cfg`, which allows to disable it when syncing the wallet.
```

Comment:
```suggestion
This involves caching some computed data in the wallet, in case a bug is discovered with this, a new configuration option `wallet_caching_disabled` will be added to `joinmarket.cfg`, which allows to disable it when syncing the wallet.
```

Reply:
> It is added, not will be added.

Yep, probably should do the other part of @MarnixCroes 's edit here, though, i.e. "in case a bug is discovered".